robottelo/hosts.py

```diff
@@ -250,6 +252,29 @@ def power_control(self, state=VmState.RUNNING, ensure=True):
             except TimedOutError:
                 raise ContentHostError('Unable to connect to host that should be running')
 
+    def download_file(self, file_url, local_path=None, file_name=None):
+        """Downloads file from given fileurl to directory specified by local_path by given filename
+        on satellite.
+
+        If remote directory is not specified it downloads file to /tmp/.
+
+        :param str file_url: The complete server file path from where the
+            file will be downloaded.
+        :param str local_path: Name of directory where file will be saved. If not
+            provided file will be saved in /tmp/ directory.
+        :param str file_name: New name of the Downloaded file else its given from file_url
+
+        :returns: Returns list containing complete file path and name of downloaded file.
+        """
+        file_name = PurePath(file_name or file_url).name
+        local_path = PurePath(local_path or '/tmp') / file_name
```

Comment:
This should be using `settings.robottelo.tmp_dir`

https://github.com/SatelliteQE/robottelo/blob/master/conf/robottelo.yaml.template#L8

Reply:
Certainly, we can go for it, but I see many places in robottelo are still using `/tmp` directly or `/var/tmp` as a hardcoded one! 

So can we handle all those places in a separate PR! For this PR moving from `/tmp` to `/var/tmp` might or not cause the problem and for testing again it might take some time !

Again this PR is not improvement PR but a quick fix PR that @adarshdubey-star brought up !

Reply:
I can accept that.